workers/unity/Packages/com.improbable.gdk.deploymentlauncher/DeploymentLauncher.cs

```diff
@@ -46,6 +49,13 @@ internal class DeploymentInfo
             public string Id;
         }
 
+        internal enum DeploymentRegionCode
+        {
+            US,
+            EU,
+            AP
```

Comment:
Can users actually create deployments here?

Reply:
I assume so, because it's referenced in the public Platform SDK docs: https://docs.improbable.io/reference/13.6/platform-sdk/api-reference/home#deployment

Reply:
Checked with Compute, looks like users can't 😢 

will update